src/main/scala/simd/Parameter.scala

```diff
@@ -14,4 +14,9 @@ object SIMDConstant {
 
   // SIMD parallelism
   def laneLen = 64
+
+  // csrManager parameters, we use 3 csr for Post-processing SIMD
+  def csrNum: Int = 3 + 1
```

Comment:
Why separate 3+1?

It would be nice to have a description what these 3+1 are 😄

Reply:
Plus 1 for status CSR!